From 27c3aab22eea3e2a7f85433e1cb92ab57aa57e27 Mon Sep 17 00:00:00 2001
From: Wyon Bi <bivvy.bi@rock-chips.com>
Date: Thu, 13 Sep 2018 09:38:09 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808: Add #address/#size-cells to dsi

DTC warnings:
arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dtb: Warning (reg_format): "reg" property in
/dsi@ffb30000/panel@0 has invalid length (4 bytes) (#address-cells == 2, #size-cells == 1)
arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dtb: Warning (avoid_default_addr_size): Relying on
default #address-cells value for /dsi@ffb30000/panel@0
arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dtb: Warning (avoid_default_addr_size): Relying on
default #size-cells value for /dsi@ffb30000/panel@0

Change-Id: I257595e10120f960c7ab6ff169302b4f2e6a68be
Signed-off-by: Wyon Bi <bivvy.bi@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 94f5ea991008..56d7baafc74c 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -887,12 +887,11 @@
 		phy-names = "mipi_dphy";
 		power-domains = <&power RK1808_PD_VIO>;
 		rockchip,grf = <&grf>;
+		#address-cells = <1>;
+		#size-cells = <0>;
 		status = "disabled";
 
 		ports {
-			#address-cells = <1>;
-			#size-cells = <0>;
-
 			port {
 				dsi_in_vop_lite: endpoint {
 					remote-endpoint = <&vop_lite_out_dsi>;
-- 
2.35.3

